Sources/SwiftDocC/Model/ParametersAndReturnValidator.swift

```diff
@@ -222,12 +222,19 @@ struct ParametersAndReturnValidator {
         var variants = DocumentationDataVariants<ReturnsSection>()
         
         var traitsWithNonVoidReturnValues = Set(signatures.keys)
-        for (trait, signature) in signatures where !signature.returns.isEmpty {
-            // Don't display any return value documentation for language representations that only return void.
-            if let language = trait.interfaceLanguage.flatMap(SourceLanguage.init(knownLanguageIdentifier:)),
-               let voidReturnValues = Self.knownVoidReturnValuesByLanguage[language],
-               signature.returns.allSatisfy({ voidReturnValues.contains($0) })
-            {
+        for (trait, signature) in signatures {
+            /// A Boolean value indicating whether the current signature returns a known "void" value.
+            var returnsKnownVoidValue: Bool {
+                if let language = trait.interfaceLanguage.flatMap(SourceLanguage.init(knownLanguageIdentifier:)),
+                   let voidReturnValues = Self.knownVoidReturnValuesByLanguage[language]
+                {
+                    return signature.returns.allSatisfy { voidReturnValues.contains($0) }
+                }
+                return false
```

Comment:
nitpick: I think it would be slightly more readable as a `guard/else` rather than an `if`
```suggestion
                guard let language = trait.interfaceLanguage.flatMap(SourceLanguage.init(knownLanguageIdentifier:)),
                   let voidReturnValues = Self.knownVoidReturnValuesByLanguage[language]
                else {
                    return false
                }
                return signature.returns.allSatisfy { voidReturnValues.contains($0) }
```

Reply:
I agree. A `guard` statements makes the early return more explicit here.